hello-strongbox-nuget/Mono.HelloWorld.nuspec

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+	<metadata>
+		<id>Strongbox.Nuget.HelloWorld</id>
```

Comment:
@sbespalov,

Could you please change this to `Org.Carlspring.Strongbox.Examples.Nuget.Mono`? (I believe this is the standard naming convention under Nuget).